metadata.rb

```diff
@@ -6,4 +6,5 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.2.0'
 
-depends 'ark'
+depends 'ark', "= 0.9.0"
+depends 'apt'
```

Comment:
Do you remember why you needed to include this? Did it cause issues for you with test kitchen? If so you can include apt as a recipe in the kitchen.yml. That way it does not preclude anyone from using redhat or centos.

Just add it to the run_list here: https://github.com/X0nic/chef-snapraid/blob/be538da3b4484cfc5efc759df46bf6b843eb509c/.kitchen.yml#L16